### PR TITLE
Fix in dl1 lesson2-sgd.ipynb

### PR DESCRIPTION
**Pull request by rraminen**
Fix for the below error:
image

Replaced a with a.detach().numpy()

### CODE REVIEW

File: nbs/dl1/lesson2-sgd.ipynb [hunk 1]
```diff
@@ -18606,7 +18606,7 @@
     "\n",
     "fig = plt.figure()\n",
     "plt.scatter(x[:,0], y, c='orange')\n",
-    "line, = plt.plot(x[:,0], x@a)\n",
+    "line, = plt.plot(x[:,0], x@a.detach().numpy())\n",
```
Comment (andresesfm) [inline nbs/dl1/lesson2-sgd.ipynb:18609]: Same thing needs to be added here:
`line.set_ydata(x@a.detach().numpy())`
